src/main/java/org/helpberkeley/memberdata/WorkflowExporter.java

```diff
@@ -40,7 +41,9 @@ public class WorkflowExporter extends Exporter {
     private final WorkflowParser parser;
     private final List<WorkflowBean> updatedBeans = new ArrayList<>();
     private final Set<String> updatedUsers = new HashSet<>();
-    private int member_limit = Constants.AVG_RUN_SIZE*10;
+    public static int getDefaultMemberLimit() { return Constants.AVG_RUN_SIZE*10; }
+    public static int member_limit = getDefaultMemberLimit();
+    public static int memberLimit() { return member_limit; } //inelegant solution?
```

Comment:
either make member_limit private or remove the memberLimit() accessor. don't need both. if it's a public variable you can just call WorkflowExporter.member_limit without an accessor